tests/cpp/test_embedding_node.cpp

```diff
@@ -0,0 +1,50 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <csrc/exceptions.h>
+#include <gtest/gtest.h>
+
+#include <fusion.h>
+#include <ops/all_ops.h>
+#include <ops/utils.h>
+#include <tests/cpp/utils.h>
+#include <tests/cpp/validator.h>
+
+namespace nvfuser {
+
+using EmbeddingTest = NVFuserTest;
+
+constexpr int64_t n = 5, s = 2;
+
+TEST_F(EmbeddingTest, EmbeddingFwdNode) {
```

Comment:
I wonder if it's a possible to add a check to verify the output of the `toString` method as well.

Reply:
I don't find verifying against a handwritten string to be very robust since that representation can change based on individual `toString` methods so I don't add it.